README.md

```diff
@@ -68,9 +68,10 @@ following keys:
 
   - `key` - the name of the cookie to use to store the token secret
     (defaults to `'_csrf'`).
-  - `path` - the path of the cookie (defaults to `'/'`).
   - any other [res.cookie](http://expressjs.com/4x/api.html#res.cookie)
-    option can be set.
+    option can be set. in particular, setting `signed: true` with your
+    server-generated secure secret hosted in `req.secret` can harden
```

Comment:
None of our tests ever set anything in `req.secret` and I have never seen anyone do that with `signed: true`. Can you elaborate on this part? Not sure I understand here why someone needs to set `req.secret` manually.

Reply:
how about I reference that to https://www.npmjs.com/package/cookie-parser#cookieparsersecret-options